charts/cortex/templates/ingester/ingester-svc.yaml

```diff
@@ -15,4 +16,38 @@ spec:
       protocol: TCP
       name: http-metrics
       targetPort: http-metrics
+    - port: {{ .Values.config.server.grpc_listen_port }}
```

Comment:
I think this is probably taken from the headless service? The statefulset needs a headless service and there is a difference in how DNS entries is created for the headless statefulset service and a normal service. The headless service will result in extra entries ala `pod-X.svcName`. I am not sure but I recall some of the discovery mechanisms in Cortex relies on this - it may be the memcached discovery though.

Nevertheless I think we should still distinguish between the headless services associated with the statefulset and the "normal" service used for ingress.